### DISCUSSION TITLE
prowler viewer project

### DISCUSSION THREAD

**Discussion by stremovsky** [prowler-cloud/prowler#3022] [General]
Hi guys

I've been cleaning up my old projects and decided to release a little tool I created to display Prowler reports.

This is a simplified Prowler JSON report viewer inspired by the Airtable design.

Here is a link:
https://github.com/stremovsky/prowler-viewer

You are welcome to check it out!

If it is possible, I would like to ask to add a link to this project in the prowler readme file.

Thank you

**Comment by jfagoagas** [comment]
That's great @stremovsky !! Actually all the projects that use Prowler are included within the `contrib` folder, which is not maintained, so you can open a PR to the repository to save your project there or to include a README within a folder in that directory or maybe pointing a folder to your repository, but we do not include links to these projects in the README file since this is just for the Prowler repository.

Thanks for using Prowler 🚀
